data/json/monstergroups/zombie_upgrades.json

```diff
@@ -18,8 +18,11 @@
       { "monster": "mon_zombie_necro", "weight": 15 },
       { "monster": "mon_zombie_necro_boomer", "weight": 12 },
       { "monster": "mon_boomer", "weight": 90 },
-      { "monster": "mon_zombie_brute", "weight": 45 },
-      { "monster": "mon_zombie_master", "weight": 3 },
+      { "monster": "mon_zombie_brute", "weight": 20 },
+      { "monster": "mon_zombie_bruiser", "weight": 30 },
+      { "monster": "mon_zombie_master", "weight": 30 },
```

Comment:
Did you intend to multiply the master's weight by 10?

Reply:
wtf how
```suggestion
      { "monster": "mon_zombie_master", "weight": 3},
```